moveit_core/robot_model/CMakeLists.txt

```diff
@@ -12,20 +13,25 @@ add_library(${MOVEIT_LIB_NAME}
   src/revolute_joint_model.cpp
   src/robot_model.cpp
   )
-set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
```

Comment:
we need to keep this feature, per @v4hn 's previous comments

Reply:
Addressed already, see https://github.com/ros-planning/moveit2/pull/10/files#diff-607eaa156ff446c97d98425f0116288cR16